backend/backend/Program.cs

```diff
@@ -25,6 +27,8 @@ public static void Main(string[] args)
 
         // Add custom services for DI
         builder.Services.AddSingleton<IAimTrainerGameService, AimTrainerGameService>();
+        builder.Services.AddSingleton<MathGameService>();
+        builder.Services.AddSingleton<MemoryGameService>();
```

Comment:
Probably labiau ateiciai, bet situs probably reiks padaryt kaip `AimTrainerGame` su interface'ais.

Reply:
Man atrodo reiks, nes yra tas DI reikalavimas. Also, jei kazkada keistume sitas dalykas labai pasitarnautu.

Reply:
Tai cia ir taip DI yra. Apart testu not sure kur pasitarnautu, bet yeah, manau reiktu prideti interfaces